src/enclave/rpcendpoint.h

```diff
@@ -2,35 +2,26 @@
 // Licensed under the Apache 2.0 License.
 #pragma once
 
-#include "http.h"
+#include "framedtlsendpoint.h"
 #include "node/rpc/jsonrpc.h"
 #include "rpcmap.h"
-#include "tlsframedendpoint.h"
 
 namespace enclave
 {
-#ifdef HTTP
-  using ServerEndpoint = HTTPEndpoint<http::ResponseHeaderEmitter>;
-#else
-  using ServerEndpoint = FramedTLSEndpoint;
-#endif
-
-  class RPCEndpoint : public ServerEndpoint
+  class RPCEndpoint : public FramedTLSEndpoint
```

Comment:
`RPCEndpoint` is quite generic. Should this be `FramedJSONRPCEndpoint` instead?

Reply:
Yes - I think a lot of the `RPC_` types should become `FramedJSONRPC_` or similar, but I'd like to save this for a separate PR.